<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPageStatistic</class>
 <widget class="QWidget" name="ViewPageStatistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>32</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Statistic</string>
       </property>
       <property name="H1" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="statisticFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="mainLabel">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Calculation resources</string>
          </property>
          <property name="white" stdset="0">
           <bool>true</bool>
          </property>
          <property name="H2" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="fromDateLabel">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>Start date</string>
              </property>
              <property name="white" stdset="0">
               <bool>true</bool>
              </property>
              <property name="H2" stdset="0">
               <bool>true</bool>
              </property>
              <property name="regular_light" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="fromDate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="toDateLabel">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="text">
               <string>End date</string>
              </property>
              <property name="white" stdset="0">
               <bool>true</bool>
              </property>
              <property name="H2" stdset="0">
               <bool>true</bool>
              </property>
              <property name="regular_light" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="toDate">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="text">
             <string>Download statistics</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="primary" stdset="0">
             <bool>false</bool>
            </property>
            <property name="secondary" stdset="0">
             <bool>true</bool>
            </property>
            <property name="lightweight_grey" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="storageLabel">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="text">
             <string>Storage</string>
            </property>
            <property name="white" stdset="0">
             <bool>true</bool>
            </property>
            <property name="H2" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadFromStorageButton">
            <property name="text">
             <string>Download statistics</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="primary" stdset="0">
             <bool>false</bool>
            </property>
            <property name="secondary" stdset="0">
             <bool>true</bool>
            </property>
            <property name="lightweight_grey" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
